docs/docs/development/documentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::note
+The documentation is written with [docusaurus](https://docusaurus.io/). You don't need install anything specific for docusaurus but it can be helpful to reference their documentation if needed.
+:::
+
+:::note
+If you are working with the documentation from within VSCode+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll n eed to restart the server when saving changes to the documentation.
```

Comment:
```suggestion
If you are working with the documentation from within VS Code+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll need to restart the server when saving changes to the documentation.
```

Use "VS Code" instead of "VSCode".

Typo: "n eed" -> "need"